common/v2/features/AddAccount/components/DeterministicWallets.tsx

```diff
@@ -92,8 +95,9 @@ class DeterministicWalletsClass extends React.PureComponent<Props, State> {
   }
 
   public render() {
-    const { wallets, network, dPaths, onCancel } = this.props;
+    const { wallets, dPaths, onCancel } = this.props;
     const { selectedAddress, customPath, page } = this.state;
+    const network = getNetworkByDPath(this.state.currentDPath) || this.props.network;
```

Comment:
Dpaths are not unique to a Network, so I don't understand why you would want to identify the network from a dPath. Please ping me on slack to talk about this.